osu.Game/Database/BeatmapDatabase.cs

```diff
@@ -25,14 +25,12 @@ public class BeatmapDatabase
         public event Action<BeatmapSetInfo> BeatmapSetAdded;
         public event Action<BeatmapSetInfo> BeatmapSetRemoved;
 
-        private BeatmapImporter ipc;
-
         public BeatmapDatabase(Storage storage, GameHost importHost = null)
         {
             this.storage = storage;
 
             if (importHost != null)
-                ipc = new BeatmapImporter(importHost, this);
+                new BeatmapImporter(importHost, this);

```

Comment:
that means that disposal/finalizer logic is not implemented, which is an issue. this reference should remain and disposal logic should be fixed.